Fix check now that fennec is pinned to 68

jcristau opened the pull request:
This doesn't *quite* work because beta doesn't get an update every day, so maybe we should get rid of the check instead, or avoid running it on weekends, to avoid finding "old" buildids.

jcristau committed:
Update fennec version and location

fennec nightly moved to mozilla-beta on 68.

jcristau committed:
Add some error handling

jcristau committed:
Fix flake8 error

marco-c commented on script.py:
```diff
@@ -3,12 +3,12 @@
 def get_nightly_version():
-    r = requests.get('https://product-details.mozilla.org/1.0/firefox_versions.json')
-    return r.json()['FIREFOX_NIGHTLY']
+    return '68.0a1'
```
We could use `alpha_version` from https://product-details.mozilla.org/1.0/mobile_versions.json

jcristau replied:
I considered that.  We'll need to update the download URL in a little while anyway when moving from beta to esr68, and I'm not sure what the version will be after esr68 branches out, so I'll keep this in mind for a later update.

jcristau committed:
Change cutoff for missing builds to 3 days instead of 2

With fennec being built off beta, we may not have uplifts (and thus
builds) for a few days, especially on weekends.

marco-c commented on script.py:
```diff
@@ -31,6 +31,7 @@ def get_socorro_buildIDs(version):
         '_results_number': 0,
         '_facets': 'build_id',
     })
+    r.raise_for_status()
```
:+1:

jcristau commented:
Updated to only alert for 3 days old builds to hopefully avoid alerting every weekend.
